### ISSUE TITLE
Emplacement de l'icône text editor

### ISSUE THREAD

**Issue by CyberTroll** [broncowdd/BoZoN#175]
Je trouve que l'emplacement de l'icône édition de texte n'est judicieusement placé (je la cherche tout le temps). Je la placerai dans le menu du répertoire courant (comme celle de création d'un répertoire) et non comme c'est le cas maintenant dans le menu du haut. Ce cernier sert plus à la gestion de l'instance de Bozon plutôt que du répertoire courant.

**Comment by broncowdd** [comment]
Oui, c'est déjà fait pour la version 3 sur laquelle je travaille ;-)
